var/spack/repos/builtin/packages/chapel/package.py

```diff
@@ -723,23 +757,57 @@ def setup_build_environment(self, env):
 
     def setup_run_environment(self, env):
         self.setup_env_vars(env)
-        env.prepend_path(
-            "PATH", join_path(self.prefix.share, "chapel", self._output_version_short, "util")
+        chpl_home = join_path(self.prefix.share, "chapel", self._output_version_short)
+        env.prepend_path("PATH", join_path(chpl_home, "util"))
+        env.set(
+            "CHPL_MAKE_THIRD_PARTY",
+            join_path(self.prefix.lib, "chapel", self._output_version_short),
+        )
+        env.set("CHPL_HOME", chpl_home)
+
+    def get_main_version_from_cmakelists(self) -> str:
+        with open(join_path(self.build_directory, "CMakeLists.txt"), "r") as f:
```

Comment:
I don't know where `build_directory` points post-install (e.g. when running Spack post-install tests).

Have you confirmed that points to `prefix/share/chapel`?